### DISCUSSION TITLE
Reset TextField to Default State with updateTextField()

### DISCUSSION THREAD

**Discussion by kyle-power** [Appsilon/shiny.fluent#21] [Q&A] [answered]
Hello again, @kamilzyla 

This time I am trying to "reset" some input fields after a button is clicked. Essentially, I just want `Dropdown` and `TextField` inputs to return to default state. My `card` is structured as the following...
```{r}
makeCard(title = "Input the Following...",
         Stack(horizontal = TRUE, tokens = list(childrenGap = 10), verticalAlign = "end",
         TextField(input_id = ns("orig"), label = "Enter Origin", placeholder = "Choose Actions to Enable"),
         TextField(input_id = ns("dest"), label = "Enter Destination", placeholder = "Choose Actions to Enable"),
         TextField(input_id = ns("station"), label = "Enter Station", placeholder = "Choose Actions to Enable"),
         PrimaryButton(input_id = ns("submit"), text = "Commit Change")
         ), size = 6, style = "max-height: 320px"
)
```
with my `server` component as
```{r}
mod_watchlist_server <- function(input, output, session){
  ns <- session$ns
  observeEvent(input$submit, {
    updateTextField(session = session, input_id = "orig", label = "Enter Origin",
                    placeholder = "Choose Actions to Enable")
    updateTextField(session = session, input_id = "dest", label = "Enter Destination", 
                    placeholder = "Choose Actions to Enable")
    updateTextField(session = session, input_id = "station", label = "Enter Station",
                    placeholder = "Choose Actions to Enable")
  })
}
```
Everything is working as expected, except for any user-defined values are remaining visible. Even after the `update*()` is sent, the input text and dropdown selection still remain unchanged and visible. Is there an argument that I can use here to "reset" each element? Leaving `value= NULL` doesn't change any behavior.

As always, thanks again so much for your help!

**Comment by marekrogala** [comment]
Hi @kyle-power. Looks like we missed the notification about your thread. We'll look into your question!

**Comment by kamilzyla** [comment, marked as answer]
Hi @kyle-power! You can pass `value = ""` to `update*()` function to clear either `TextField` or `Dropdown`.

**Reply by kyle-power** [reply]
I swear I tried that... but I guess not. Thanks for all the help!
